drivers/hmis/app/models/hmis/access_group.rb

```diff
@@ -135,7 +124,7 @@ def associated_by(associations:)
           ]
         end
       when :project_access_group
-        project_access_groups.preload(:projects).map do |pag|
+        project_access_access_.preload(:projects).map do |pag|
```

Comment:
I'm getting errors here, this isn't defined

Reply:
See below. If you're adding back the `project_access_groups` scope in your branch, probably best to just fix this in your branch too. it should be `project_access_groups`

Reply:
its fixed on my branch